contracts/potlock/src/potlock_admin_interactions.rs

```diff
@@ -0,0 +1,111 @@
+use crate::potlock_storage::{self, PotlockId, ProjectId, Status};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub type ProjectPercentage = MultiValue2<usize, u64>;
+pub const MAX_PERCENTAGE: u64 = 10_000; // 100%
+
+#[multiversx_sc::module]
+pub trait PotlockAdminInteractions:
+    potlock_storage::PotlockStorage + multiversx_sc_modules::only_admin::OnlyAdminModule
+{
+    #[only_admin]
+    #[endpoint(changeFeeForPots)]
+    fn change_fee_for_pots(&self, token_identifier: TokenIdentifier, fee: BigUint) {
+        require!(
+            token_identifier.is_valid_esdt_identifier(),
+            "Invalid token provided"
+        );
+        require!(
+            token_identifier.is_valid_esdt_identifier() && fee.ge(&BigUint::zero()),
+            "Invalid token identifier or amount is 0"
+        );
+        self.fee_token_identifier().set(&token_identifier);
+        self.fee_amount().set(fee);
+    }
+
+    #[only_admin]
+    #[endpoint(acceptPot)]
+    fn accept_pot(&self, potlock_id: PotlockId) {
+        self.require_potlock_exists(potlock_id);
```

Comment:
I would add an extra check here, that the potlock is inactive. So there is no chance for a double call of this endpoint.